lib/components/dropdown-item.vue

```diff
@@ -1,31 +1,21 @@
 <template>
     <b-link class="dropdown-item"
             role="menuitem"
```

Comment:
In BS V4.alpha.6, dropdown items can also be a button: http://v4-alpha.getbootstrap.com/components/dropdowns/#menu-items

I winder if this could be conditional, and if no `to` or `href` set, then a `b-btn` is used.

```html
<b-link v-if="href || to" ...>
   <slot></slot>
</b-link>
<b-btn v-else class="dropdown-item" @click="$emit('click', $event)">
  </slot>
</b-btn>
```